urgent-care/src/main/resources/application.properties

```diff
@@ -1,15 +1,18 @@
 server.port=8090
 urgent-care.url=unset
 urgent-care.base-path=fhir/v0/r4
-queenelizabeth.url=unset
-health-check.patient-id=unset
-identityservice.url=unset
 
-security.require-ssl=true
+# Dummy values to run locally as dev.
```

Comment:
This is me not knowing exactly how the runtime population of these properties works (pulling from S3), but does whatever _that_ is expect for these values to be `unset` to actually set them? Or will it just override whatever is here as long as it's in the S3 file?

Also just curious if there is any existing patterns on the project of using spring profiles to load environment specific properties (`application-dev.properties`).

Reply:
As long as properties are set in s3 they will override what is in the project properties.  Otherwise there is potential for bleed through.  I might be wrong, but as far I as I know we don't use spring profiles :-(  These changes allowed me to clone, build, and run locally to test without having to also download the du, decrypt, and locally edit properties to get the application to run.  Maybe Alec or Dom can clarify if there is something I am missing.

Reply:
So the way we do it on other projects is to use a spring profile.  We have all of those listed as unset and have a local 'application-dev.properties' that developers run on their local machines.  Unfortunately the process for passing that information around is manual :(

If you throw in '-Dspring.profiles.active=dev' in your development environment it should automatically use the application-dev.properties instead.  The reason for this is some application-dev.properties have usernames and passwords for dev endpoints

Reply:
My two cents: I'd rather commit to either this being the defacto dev properties file, or having this be useless besides telling you property names to fill. Having only some properties set to dev worries me that we'll miss some weird edge case bleed through where we only set 4 of those ssl properties in the DU, and then one of these dev ones gets slipped in.

My ideal would be checking in real dev properties file alongside a useless one since I don't think anything we'd put in it would be sensitive. I'm sure there are good reasons not to go that route (like standardization with other projects) though.

Reply:
I moved the properties to dev profile.

Reply:
I started doing something with this and realizing I didn't really like having the useless application.properties file as much as I thought. Switched back to what was there before with ee header information to show those need to be set.